libraries/api/discussion_helper.cpp

```diff
@@ -85,14 +101,137 @@ namespace golos { namespace api {
             return database_;
         }
 
+        comment_api_object create_comment_api_object(const comment_object& o) const;
+
         discussion get_discussion(const comment_object& c, uint32_t vote_limit) const;
 
+        void fill_comment_content(const golos::chain::database& db, const comment_object& co, comment_api_object& cao);
+
+        void fill_comment_api_object(const comment_object& o, discussion& d) const;
+
     private:
         golos::chain::database& database_;
         std::function<void(const golos::chain::database&, const account_name_type&, fc::optional<share_type>&)> fill_reputation_;
         std::function<void(const golos::chain::database&, discussion&)> fill_promoted_;
+        std::function<void(const golos::chain::database&, const comment_object&, comment_api_object&)> fill_comment_content_;
     };
 
+// create_comment_api_object 
+    comment_api_object discussion_helper::create_comment_api_object(const comment_object & o) const {
+        return pimpl->create_comment_api_object(o);
+    }
+
+    comment_api_object discussion_helper::impl::create_comment_api_object(const comment_object & o) const {
```

Comment:
Refactor this to `fill_comment_api_object()` and `create_comment_api_object()` (`create_comment_api_object()` should call `fill_comment_api_object()`.

Reply:
Done